### DISCUSSION TITLE
Update operator-sdk version

### DISCUSSION THREAD

**Discussion by turbaszek** [kedacore/keda#1027] [General] [answered]
Currently, we use:
https://github.com/kedacore/keda/blob/9341259967bb2b657e336630431e1270b6f765e4/tools/build-tools.Dockerfile#L56

Version v1.0.0 is available since few days ago. Should we upgrade? There are some breaking changes. 
https://github.com/operator-framework/operator-sdk/releases/tag/v1.0.0

**Comment by tomkerkhove** [comment, marked as answer]
Already done: https://github.com/kedacore/keda/blob/v2/tools/build-tools.Dockerfile#L58

**Reply by tomkerkhove** [reply]
But thanks for the suggestion!

**Reply by turbaszek** [reply]
Oh, thanks! I totally missed the information that v2 is the main upstream 👍
